migrations/Migration20240816120248.ts

```diff
@@ -1,13 +1,9 @@
 import { Migration } from '@mikro-orm/migrations';
 
 export class Migration20240816120248 extends Migration {
-    async up(): Promise<void> {
-        this.addSql('alter table "rolle" add column "ist_technisch" boolean not null default false;');
-
-        this.addSql("update rolle set ist_technisch = true where name = 'Migration';");
+    public up(): void {
+        this.addSql("UPDATE service_provider SET url = 'https://school-sh.qs.schule-sh.de/' WHERE name = 'School-SH';");
```

Comment:
Shouldn't this also be changed in the seeding-data? (For fresh deployments on cluster, where migrations are applied before seeding)